src/main/java/org/takes/facets/fallback/FbStatus.java

```diff
@@ -43,8 +40,10 @@
  * <p>The class is immutable and thread-safe.
  *
  * @author Yegor Bugayenko (yegor@teamed.io)
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
  * @version $Id$
  * @since 0.13
+ * @checkstyle ClassDataAbstractionCouplingCheck (500 lines)
```

Comment:
@ekondrashev are you sure we even need this suppression? it looks to me that you actually reduced the coupling (the imports above were reduced)